app/controllers/component-tree.js

```diff
@@ -298,19 +301,19 @@ export default Controller.extend({
     /**
      * Scrolls the main page to put the selected element into view
      */
-    scrollToElement(elementId) {
+    scrollToElement(objectId) {
       this.get('port').send('view:scrollToElement', {
-        elementId
+        elementId: objectId // TODO: what?
```

Comment:
@nummi did we come to a consensus on the elementId vs objectId thing here?

Reply:
Not in this specific case. This seems like a boo boo that happens to work (most of the time?).

Reply:
Gotcha. Perhaps @rwjblue could jump on a quick call to discuss this sometime this week? I want to make sure we get all the `objectId` vs `elementId` stuff right and try to future proof and get things ready for glimmer components and such.

Reply:
ya, happy to chat in higher bandwidth